### DISCUSSION TITLE
제페토 캐릭터 컨트롤러가 항상 보이도록

### DISCUSSION THREAD

**Discussion by smlee2952** [naverz/zepeto-studio-kor#1206] [5. Q&A]
제페토 캐릭터 컨트롤러가 Play 시에는 나타나 있는데 터치하고 손을 떼는 순간 캐릭터 컨트롤러가 숨겨지고 다시 터치 시 컨트롤러가 나오기를 반복하는데 항상 캐릭터 컨트롤러가 나타나있게 하는 방법은 없나요?

**Comment by zlmrzlo** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/548
직접 만들어 보시는 것은 어떨까요? 인터넷으로 검색하면 가상패드 만드는 방법은 나와있습니다.
